phase0/CRC Design/GameManager.txt

```diff
@@ -0,0 +1,12 @@
+Class name: GameManager
```

Comment:
I don't think we need a gamemanager class cuz currently the player class manages the gamemanager

Reply:
Unless we want to allow multiple games to run at once or something I don't think we need it